assets/css/v2/pre_fare/viewport.scss

```diff
@@ -1,10 +1,20 @@
+$panel-width: 1080px;
+
+.screen-container {
+  // width of a single Pre-Fare display unit; "duo" installations are two of
+  // these side-by-side
+  --panel-width: #{$panel-width};
+}
+
+.simulation {
+  // width of a single Pre-Fare display unit scaled down by 17.92%
+  --panel-width: #{$panel-width * 0.1792};
```

Comment:
This seems like a more fragile solution than what we had before, and I wonder if we need to do it this way (encoding the percentage scale-down in two different places). I'm also confused as to why this specific change fixes the flex zone items, since this doesn't touch anything outside the `.simulation` element, which only includes the "main" screen view. Maybe you can walk me through it at some point today?